Dockerfile

```diff
@@ -13,9 +13,10 @@ WORKDIR /home/user
 
 # Add the ssh key from local dir to container dir.
 # ADD github /home/user/.ssh/id_rsa
+# ADD known_hosts /home/user/.ssh/known_hosts
 
 RUN \
-    dnf install -y --setopt=tsflags=nodocs git python38 python3-pip gcc redhat-rpm-config python3-devel which gcc-c++ &&\
+    dnf install -y --setopt=tsflags=nodocs git libgit2 python38 python36 python3-pip gcc redhat-rpm-config python3-devel which gcc-c++ &&\
```

Comment:
Try with:

```suggestion
    dnf install -y --setopt=tsflags=nodocs git libgit2-devel python38 python36 python3-pip gcc redhat-rpm-config python3-devel which gcc-c++ &&\
```

Reply:
Okay sure :+1:

Reply:
pip 9.0.1 can install pygit2 inside the venv
however the pipenv run still fails for fasttext - 
```
  File "/home/user/.local/share/virtualenvs/user-5PivF_Od/lib/python3.6/site-packages/pipenv/patched/notpip/_internal/operations/prepare.py", line 294, in prepare_linked_requirement
    abstract_dist.prep_for_dist(finder, self.build_isolation)
  File "/home/user/.local/share/virtualenvs/user-5PivF_Od/lib/python3.6/site-packages/pipenv/patched/notpip/_internal/operations/prepare.py", line 127, in prep_for_dist
    self.req.run_egg_info()
  File "/home/user/.local/share/virtualenvs/user-5PivF_Od/lib/python3.6/site-packages/pipenv/patched/notpip/_internal/req/req_install.py", line 474, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/home/user/.local/share/virtualenvs/user-5PivF_Od/lib/python3.6/site-packages/pipenv/patched/notpip/_internal/utils/misc.py", line 705, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pipenv.patched.notpip._internal.exceptions.InstallationError: Command "python setup.py egg_info" failed with error code 1 in /tmp/tmpnlbed0tebuild/fasttext/

2020-09-21 18:10:45,988  43 WARNING  kebechet.managers.update.update:723: Failed to update dependencies to their latest version, reporting issue
2020-09-21 18:10:46,800  43 INFO     thoth.sourcemanagement.sourcemanagement:86: Issue already noted on upstream with title #Failed to update dependencies to their latest version
```

Reply:
Sadly, the log is not that helpful. Does pip install work in a virtual environment?

```
python3.6 -m venv venv
source venv/bin/activate
pip install --force-reinstall pip==9.0.1  # or replicate the virtual env used
pip install fasttext
```

Reply:
```
bash-5.0$ ls
CHANGELOG.md  MANIFEST.in  Pipfile	 README.rst  config  kebechet	   kebechet.egg-info  openshift  requirements.txt  test.py
Dockerfile    OWNERS	   Pipfile.lock  app.sh      github  kebechet-cli  known_hosts	      playbooks  setup.py	   workflows
bash-5.0$ cd /tmp
bash-5.0$ python3.6 -m venv venv
bash-5.0$ source venv/bin/activate
(venv) bash-5.0$ pip install --force-reinstall pip==9.0.1  # or replicate the virtual env used
Collecting pip==9.0.1
  Downloading https://files.pythonhosted.org/packages/b6/ac/7015eb97dc749283ffdec1c3a88ddb8ae03b8fad0f0e611408f196358da3/pip-9.0.1-py2.py3-none-any.whl (1.3MB)
     |████████████████████████████████| 1.3MB 1.1MB/s 
Installing collected packages: pip
  Found existing installation: pip 19.3.1
    Uninstalling pip-19.3.1:
      Successfully uninstalled pip-19.3.1
Successfully installed pip-9.0.1
(venv) bash-5.0$ pip install fasttext
Collecting fasttext
  Downloading https://files.pythonhosted.org/packages/f8/85/e2b368ab6d3528827b147fdb814f8189acc981a4bc2f99ab894650e05c40/fasttext-0.9.2.tar.gz (68kB)
    100% |████████████████████████████████| 71kB 955kB/s 
Requirement already satisfied: pybind11>=2.2 in ./venv/lib/python3.6/site-packages (from fasttext)
Requirement already satisfied: setuptools>=0.7.0 in ./venv/lib/python3.6/site-packages (from fasttext)
Collecting numpy (from fasttext)
  Downloading https://files.pythonhosted.org/packages/b8/e5/a64ef44a85397ba3c377f6be9c02f3cb3e18023f8c89850dd319e7945521/numpy-1.19.2-cp36-cp36m-manylinux1_x86_64.whl (13.4MB)
    100% |████████████████████████████████| 13.4MB 87kB/s 
Installing collected packages: numpy, fasttext
  Running setup.py install for fasttext ... done
Successfully installed fasttext-0.9.2 numpy-1.19.2
You are using pip version 9.0.1, however version 20.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Reply:
fasttext does get install inside the venv without any issues